roles/logging/templates/etc/rsyslog.d/60-remote-syslog.conf

```diff
@@ -0,0 +1,47 @@
+$WorkDirectory {{ logging.syslog_forwarding.work_directory }}
+
+{% if logging.syslog_forwarding.action_resume is defined %}
+{% if logging.syslog_forwarding.action_resume.retry_count is defined %}
```

Comment:
you should be able to loop through these using `with_dict` rather than an if statement for each.